### ISSUE TITLE
Best classification results in ModelNet40

### ISSUE THREAD

**Issue by LiuNull** [erikwijmans/Pointnet2_PyTorch#44]
I have noticed that you had a closed issue about best results in ModelNet40.
You said before the changing, your results was only 0.X% accuracy gap with the paper. But Now I run the code with 8 gpus, I only got 0.9023% for ModelNet40. I use the default hyper-parameters. So would you please tell me what's the exact accuracy you got and why I still can't matche the performance from the paper? Thx!

**Comment by erikwijmans** [comment]
2 things that could be causing this discrepancy:

1. You need to change to use 10k points instead of the default to match the results in the paper. 
2. Batch norm is known to scale poorly in performance to multiple GPUs. Either change the BN layers to SyncBN (nvidia Apex can do this), or use the smallest number of GPUs you can.

**Comment by imankgoyal** [comment]
@erikwijmans Thanks for the implementation. @LiuNull @erikwijmans Were you able to replicate the results in the paper? I just wanted to make sure so as to have fair benchmarking? thanks

**Comment by sheshappanavar** [comment]
@LiuNull or @erikwijmans Please let me know what script you used to test the results. I have trained using the command mentioned in the readme file. However, I didn't find a script to test/evaluate the trained model. Any help regarding this is much appreciated. TIA

**Comment by sheshappanavar** [comment]
> I have noticed that you had a closed issue about best results in ModelNet40.
> You said before the changing, your results was only 0.X% accuracy gap with the paper. But Now I run the code with 8 gpus, I only got 0.9023% for ModelNet40. I use the default hyper-parameters. So would you please tell me what's the exact accuracy you got and why I still can't matche the performance from the paper? Thx!

90.23% is for MSG? did you use --num_votes during the evaluation ?
